eng/pipelines/jobs/build-test.yml

```diff
@@ -31,6 +31,10 @@ jobs:
     architecture: ${{ parameters.architecture }}
 
     variables:
+    - _PublishAfterBuildArgs: '/p:PublishProjectsAfterBuild=true /p:SkipPlatformNeutralPublish=true'
+    - ${{ if and(eq(parameters.targetRid, 'win-x64'), eq(parameters.configuration, 'Release')) }}:
+      - _PublishAfterBuildArgs: '/p:PublishProjectsAfterBuild=true'
```

Comment:
Just making sure I'm understanding this correctly, is this implicitly equivalent to `- _PublishAfterBuildArgs: '/p:PublishProjectsAfterBuild=true /p:SkipPlatformNeutralPublish=false'`

Reply:
Sure, you can think of it that way. The `SkipPlatformNeutralPublish` usage is checking if it's exactly set to `true`, so any other value would cause it to not skip.